packages/remix-cloudflare-workers-esm/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# `@remix-run/cloudflare-workers-esm`
```

Comment:
Admittedly not sure what to do with `CHANGELOG.md` in a new package, but I assume the release process solves this.